---
float64[] x
float64[] y
float64[] s
float64[] phi_f
float64[] curvature
float64[] g_c
float64[] dx
float64[] dy
float64[] ddx
float64[] ddy